src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/filters/util/toolbar-extension-handler.js

```diff
@@ -28,7 +28,7 @@ define(['app-config'], function(AppConfig) {
          * @param filterResultSelector {String}
          */
         extendToolbar = function(context, toolbarItems, toolbarInstanceName, dataGridInstanceName, filterResultSelector) {
-            if (!!context && !!toolbarItems && !!dataGridInstanceName) {
+            if (!!context && !!toolbarItems && !!dataGridInstanceName && !!this.config.contexts[context]) {
```

Comment:
To display the filter button (and do some other filter-related stuff) only if the context is also defined in the configuration. I stumbled upon this problem when wanted to tested the listbuilder without the filters.